### ISSUE TITLE
Replace fruit collection sounds

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#228]
The sounds that are played when collecting the powerups — which will be berries after the merge of #227 — still sounds like SMB. They should be replaced by more original ones.

Vale,
Quintus

**Comment by datahead8888** [comment]
Is this still a blocker task?  If so, it should be labeled with the release-2.0.0 milestone.

**Comment by Quintus** [comment]
Hm. I think we can go well without it, but we should target this sometime.

Vale,
Quintus

**Comment by datahead8888** [comment]
@sydneyjd - I know you were asking on this.
If you can find some sounds on a site with permissive licenses, we could get this knocked out for the release.
Bugsbane suggested http://freesound.org above.

**Comment by sydneyjd** [comment]
I was? Im sorry, i guess i didn't realize i was.

**Comment by datahead8888** [comment]
I think it was in IRC.  It can be hard to remember conversations, though.  I just thought I'd show you the ticket and resource websites.

**Comment by Bugsbane** [comment]
> couldn’t come up with something that sounded like just squashing some soft fruit

I would take a less literal and more expressionistic approach, in other words, don't look for something that "sounds like fruit", look for something that "sounds like achievement" etc, eg:

http://freesound.org/people/cameronmusic/sounds/138419/
http://freesound.org/people/Andromadax24/sounds/186719/
http://freesound.org/people/manden1/sounds/256112/

Remember, players are going to hear this sound 100's/1000's of times, so it needs to be fairly generic and not something that really "stands out", or they'll be hating it by coin number 3,276...

**Comment by Bugsbane** [comment]
This has been done if I'm not mistaken. When I last played the game it didn't sound like Mario coins any more... Should it be marked as closed?

**Comment by Bugsbane** [comment]
Here's a basic attempt. Hey, I'm more of a graphics than a sound guy, but at least this one's legal. :)
[Fruit power up sound - OGG](http://crowdfundingdojo.com/temp/chime-chomp.ogg)

I can supply an audacity AUP file for sources if desired (really we should collect these, just as we collect Inkscape SVG's for graphics)

**Comment by datahead8888** [comment]
Honestly, I have a bit of trouble seeing this in the game for power ups, but it depends on what type of sound theme we're going for.  I'm open to other views, though.

**Comment by Bugsbane** [comment]
Yeah, I don't love it that much myself. I figured I'd give it a shot, even if just to spur conversation about what it should be.

**Comment by datahead8888** [comment]
Even if it doesn't get used for power ups, it can certainly be used somewhere in the game.  Thanks for submitting this, either way :)

**Comment by Quintus** [comment]
@Luiji, @brianvanderburg2 I’m busy with university stuff. Could you two review these and help @datahead8888?

Valete,
Quintus

**Comment by Luiji** [comment]
@Quintus Bookmarked.

**Comment by Quintus** [comment]
@kirbyfan64 are you up for creating some fruit collection replacement sounds? If not, I will pursue the existing (non-unique) alternatives mentioned in this ticket so this finally gets resolved.

Valete,
Quintus

**Comment by datahead8888** [comment]
@kirbyfan64 - would you be up for this?
